src/framework/preferences/css/sass/EnhanceInputs_base.scss

```diff
@@ -0,0 +1,6 @@
+.fl-input-enhanced {
+    --fl-enhance-factor: 1.25;
```

Comment:
This value seems to be used in just one place. Is there a use case for adjusting it somehow?
Given it appears to be tied to the next value, if we keep it, could be named --fl-enhance-font-size-factor